library/core/src/ptr/const_ptr.rs

```diff
@@ -388,6 +388,12 @@ impl<T: ?Sized> *const T {
     ///   bounds of that allocated object. In particular, this range must not "wrap around" the edge
     ///   of the address space.
     ///
+    /// * Note that the special case of "one-past-the-end" pointers is explicitly allowed: a pointer
```

Comment:
We are not using "one-past-the-end" terminology in Rust (I think it's terrible terminology).

Also, as far as I know all the existing docs already handle this properly.